paramiko/auth_handler.py

```diff
@@ -566,7 +604,6 @@ def _parse_userauth_failure(self, m):
     def _parse_userauth_banner(self, m):
         banner = m.get_string()
         self.banner = banner
-        lang = m.get_string()
```

Comment:
Yea this feels like an oops to me, I think probably best to be consistent with the change in `kex_gss.py`, where we just do a discarded `m.get_string()`.